src/main/bluesky/parser.py

```diff
@@ -37,53 +37,62 @@ def calculate_bearing(lat1, lon1, lat2, lon2):
     return round(0 if np.isnan(bearing) else bearing, 2)
 
 
-def write_scene_file(filename, combined_df):
+def write_scene_file(filename, combined_df, total_time):
     with open(filename, 'w') as file:
+        current_time = 0
+        time_step = int(total_time) / len(combined_df)
+
         for index, row in combined_df.iterrows():
             scenetext = ""
             lat, lon = get_airport_info(row["DEST"], "lat"), get_airport_info(row["DEST"], "lon")
             ehamLat, ehamLon = 52.309, 4.764
             stack = row.get("STACK", None)
-            # check if stack is not empty and set stack and brng
+
+            # Calculate time stamp
+            current_time += time_step
+            hours = int(current_time // 3600)
+            minutes = int((current_time % 3600) // 60)
+            seconds = current_time % 60
+            time_stamp = f"{hours:02d}:{minutes:02d}:{seconds:.2f}"
+
             if lat is not None and lon is not None and stack in ["RIVER", "SUGOL", "ARTIP"]:
                 lat, lon = {
                     "RIVER": (51.912, 4.132),
                     "SUGOL": (52.525, 3.967),
                     "ARTIP": (52.511, 5.569)
                 }.get(stack, (lat, lon))
                 brng = calculate_bearing(lat, lon, ehamLat, ehamLon)
-                # else brng is calculated from origin
             else:
                 brng = calculate_bearing(ehamLat, ehamLon, lat, lon)
+
             flight_altitude = int(float(row["RFL"]))
-            # if origin is Netherlands, set the origin as origin and delete at a distance of 30
+
             Nederland = get_dutch_airports()
             if row['ADEP'] in Nederland:
                 scenetext = (
-                    f"00:00:00.00>CRE {row['CALLSIGN']} {row['ICAO_ACTYPE']} {row['ADEP']} {brng} "
-                    f"FL{flight_altitude} {row['TAS']}\n"
-                    f"00:00:00.00>DEST {row['CALLSIGN']} {row['DEST']}\n"
-                    f"00:00:00.00>{row['CALLSIGN']} ATDIST {row['ADEP']} 30 DEL {row['CALLSIGN']}\n"
+                    # f"{time_stamp}>CRE {row['CALLSIGN']} {row['ICAO_ACTYPE']} {row['ADEP']} {brng} "
+                    # f"FL{flight_altitude} {row['TAS']}\n"
```

Comment:
should removed unused code before merging into main

Reply:
yeah i know, this doesnt make a scene file but just checks if inbound or outbound for easy visual if it works, the checked code is the one that makes the scene that i will uncheck at the end